examples/cvector-generator/cvector-generator.cpp

```diff
@@ -2,8 +2,7 @@
 #include "common.h"
 #include "llama.h"
 #include "ggml.h"
-#include "pca.hpp"
-#include "mean.hpp"
+#include "vanilla_pca.hpp"
```

Comment:
Maybe you can just replace `pca.hpp` with your implementation? (I don't get what does "vanilla" mean in this context)